src/locale/defaults.h

```diff
@@ -734,7 +734,7 @@
 #endif  // D_STR_MESGDESC
 #ifndef D_STR_IRRECVDUMP_STARTUP
 #define D_STR_IRRECVDUMP_STARTUP \
-    "IRrecvDumpV2 is now running and waiting for IR input on Pin %d"
+    "IRrecvDump is now running and waiting for IR input on Pin %d"
```

Comment:
Change // IRrecvDumpV2 at line 725 as well?

See similar comment on zh-CN.h, but this applies to all files

Reply:
Done.